index.html

```diff
@@ -135,13 +135,35 @@
       <section class="promo">
         <div class="promo-text">the right ingredients for the right food</div>
 
-        <img src="../img/divider.png" alt="divider" />
+        <img src="../img/divider.png" alt="divider" class="divider-lg" />
 
         <div class="promo-buttons">
           <button class="btn btn-black">Book a table</button>
           <button class="btn btn-white">See the menu</button>
         </div>
       </section>
+
+      <section class="about">
+        <figure class="about-content">
+          <h2 class="about-heading margin-bottom-sm">Just the right food</h2>
+          <img
+            src="../img/divider-gold.png"
+            alt="divider"
+            class="divider-md margin-bottom-lg"
+          />
+          <p class="about-text margin-bottom-xl">
+            If you’ve been to one of our restaurants, you’ve seen – and tasted –
+            what keeps our customers coming back for more. Perfect materials and
+            freshly baked food, delicious Lambda cakes, muffins, and gourmet
+            coffees make us hard to resist! Stop in today and check us out!
+          </p>
+          <img src="../img/cook.png" alt="cook" class="cook" />
```

Comment:
Alt text should draw mind image for those who don't the picture
```suggestion
          <img src="../img/cook.png" alt="Chef decorates the dish" class="cook" />
```